includes/class-amp-http.php

```diff
@@ -490,4 +492,23 @@ public static function filter_comment_post_redirect( $url, $comment ) {
 
 		return null;
 	}
+
+	/**
+	 * Get the Content-Type for the response.
+	 *
+	 * @since 1.2
+	 *
+	 * @return string Content type.
+	 */
+	public static function get_response_content_type() {
+		$content_type = ini_get( 'default_mimetype' );
+		foreach ( headers_list() as $header ) {
+			list( $name, $value ) = explode( ':', $header, 2 );
+			if ( 'content-type' === strtolower( $name ) ) {
+				$content_type = trim( $value );
```

Comment:
This could just be a `return trim( $value )` without the `break`, no?

Reply:
It could, but I personally prefer to have only one `return` per function, for consistency and for future potential in case a filter is added. (Not that a filter would be added here.)